src/ape/managers/config.py

```diff
@@ -168,12 +176,22 @@ def _plugin_configs(self) -> Dict[str, PluginConfig]:
                 "Plugins may not be installed yet or keys may be mis-spelled."
             )
 
-        self._plugin_configs_by_project[project_name] = configs
+        self._cached_configs[project_name] = configs
         return configs
 
     def __repr__(self):
         return f"<{self.__class__.__name__} project={self.PROJECT_FOLDER.name}>"
 
+    def __str__(self) -> str:
+        """
+        The JSON-text version of the project config data.
+
+        Returns:
+            str
+        """
+
+        return self.json()
```

Comment:
Why is this returning json for str method?

Reply:
I don't know, I didn't write this method, it was just moved it near the `__repr__` in effort to clean this up a little.

The type of `json()` return is a `str` though.

I am not sure what uses this.

Reply:
yeah... this seems like a good candidate for optimizing out

Reply:
💨